content/node-runners/postinstall/index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Post install configuration
+weight: 10
+---
+
+### Startup configuration
+The installed DEB package will be starting a node service automatically.
+
+You can check details using the following command:
+
+```shell
+sudo systemctl status mysterium-node.service
+```
+
+You can change the configuration of the service via `/etc/default/mysterium-node` file:
+
+```shell
+root@raspberrypi:~# cat /etc/default/mysterium-node
+# Define additional args for `myst` service (see `myst --help` for full list)
+CONF_DIR="--config-dir=/etc/mysterium-node"
+RUN_DIR="--runtime-dir=/var/run/mysterium-node"
+DATA_DIR="--data-dir=/var/lib/mysterium-node"
+DAEMON_OPTS="--tequilapi.address=0.0.0.0"
+SERVICE_OPTS="openvpn"
+```
+
+List of available options can be found using a help commands: `myst --help` or `myst service --help`.
+
+For example, if you want to start only OpenVPN service on 1190 port and serve Mysterium verified consumer only, change the `SERVICE_OPTS=` line to the following:
+
+```shell
+SERVICE_OPTS="--openvpn.port=1190 --access-policy.list=mysterium openvpn"
```

Comment:
Do we still use this 1190 port?
Could we describe what to do with Wireguard as it is our main protocol?